coral/src/app/layout/SidebarNavigation.tsx

```diff
@@ -0,0 +1,113 @@
+import { Box, Flexbox, Icon } from "@aivenio/design-system";
+import database from "@aivenio/design-system/dist/src/icons/database";
+import codeBlock from "@aivenio/design-system/dist/src/icons/codeBlock";
+import layoutGroupBy from "@aivenio/design-system/dist/src/icons/layoutGroupBy";
+import code from "@aivenio/design-system/dist/src/icons/code";
+import people from "@aivenio/design-system/dist/src/icons/people";
+import list from "@aivenio/design-system/dist/src/icons/list";
+import cog from "@aivenio/design-system/dist/src/icons/cog";
+import data from "@aivenio/design-system/dist/src/icons/console";
+import classes from "src/app/layout/SidebarNavigation.module.css";
+
+const originLocationKlawAngular = window.location.origin;
+function createSidebarListItem({
+  icon,
+  href,
+  linkText,
+  active = false,
+}: {
+  icon: typeof data;
+  href: string;
+  linkText: string;
+  active?: boolean;
+}) {
+  return (
+    <Flexbox
+      htmlTag={"li"}
+      alignItems={"center"}
+      marginBottom={"l1"}
+      colGap={"l1"}
+      className={
+        active
+          ? classes.primaryNavigationLinkActive
+          : classes.primaryNavigationLink
+      }
+      paddingLeft={"l2"}
+    >
+      <Icon icon={icon} />
```

Comment:
I think the icon should be part of the link. If I hover the icon the background changes, but cursor does not (and it is no clickable).

Reply:
Oh, great catch, I missed checking for this.

I'll add a style to the Icon to get the spacing right for now. The Icon component does not have a prop for that, but I think that can be something the DS could add?

Reply:
I believe Icon should not care about its padding / margins. I would wrap the icon + link text into Flexbox or something. That makes the DOM slightly more complex, but I find that as the best option.